test/framework/easyblock.py

```diff
@@ -611,7 +611,7 @@ def test_exclude_path_to_top_of_module_tree(self):
             for imkl_dep in excluded_deps:
                 tup = (imkl_dep, modfile_path, modtxt)
                 failmsg = "No 'module load' statement found for '%s' not found in module %s: %s" % tup
-                self.assertFalse(re.search("module load %s" % imkl_dep, modtxt), failmsg)
+                self.assertFalse(re.search('module load "%s"' % imkl_dep, modtxt), failmsg)
```

Comment:
this is wrong, twice... :-)

* a `module load` statement in a module file in Tcl syntax will not include quotes
* this check is useful in case the module file is in Lua syntax, where it would be `load("foo")`